sale_crm_medical_prescription/wizards/medical_lead_wizard.py

```diff
@@ -28,64 +24,57 @@ class MedicalLeadWizard(models.TransientModel):
         required=True,
         default=lambda s: s._compute_default_pharmacy(),
     )
-    split_orders = fields.Selection([
-        ('partner', 'By Customer'),
-        ('patient', 'By Patient'),
-        ('all', 'By Rx Line'),
-    ],
+    split_orders = fields.Selection(
+        string='Split Orders',
+        selection=[
+            ('patient', 'By Patient'),
+        ],
         default='patient',
         required=True,
         help='How to split the new orders',
     )
 
-    def _compute_default_session(self, ):
+    def _compute_default_session(self):
         return self.env['medical.prescription.order.line'].browse(
             self._context.get('active_ids')
         )
 
-    def _compute_default_pharmacy(self, ):
+    def _compute_default_pharmacy(self):
         default_order_lines = self._compute_default_session()
         if len(default_order_lines):
             return default_order_lines[0].prescription_order_id.partner_id
 
     @api.multi
-    def action_create_leads(self, ):
-
+    def action_create_leads(self):
         order_map = defaultdict(list)
+
         for rx_line in self.prescription_line_ids:
-            if self.split_orders == 'partner':
-                raise NotImplementedError(_(
-                    'Patient and Customers are currently identical concepts.'
-                ))
-            elif self.split_orders == 'patient':
-                order_map[rx_line.patient_id].append(rx_line)
-            else:
-                order_map[None].append(rx_line)
+            order_map[rx_line.patient_id].append(rx_line)
 
-        lead_obj = self.env['crm.lead']
+        lead_obj = self.env['crm.lead'].browse()
         lead_ids = lead_obj
-        for patient_id, order in order_map.items():
 
+        for patient_id, rx_orders in order_map.items():
             partner_id = patient_id.partner_id
-            order_lines = [(4, l.id, 0) for l in order]
+            rx_order_lines = [(4, rx_order.id, 0) for rx_order in rx_orders]
+
             lead_ids += lead_obj.create({
                 'partner_id': partner_id.id,
                 'email_from': partner_id.email,
                 'phone': partner_id.phone,
                 'pharmacy_id': self.pharmacy_id.id,
-                'prescription_order_line_ids': order_lines,
+                'prescription_order_line_ids': rx_order_lines,
                 'is_prescription': True,
-                'name': ', '.join([l.name or '' for l in order])
+                'name': ', '.join(rx_order.name for rx_order in rx_orders),
```

Comment:
The test for this method is failing because of the order of `rx_orders`. Perhaps you could `rx_orders.sorted(key=lambda r: r.name)`?